### ISSUE TITLE
EPUB 3 Feature Support

### ISSUE THREAD

**Issue by sirespin** [readium/readium-js#198]
Good Morning,

Does anyone know where I can find the EPUB 3 features that readium supports?

I have found http://epubtest.org/features/, but they are not updated.

Thank you so much.

**Comment by danielweck** [comment]
Hello, in `epubtest.org` ( http://epubtest.org/testsuite/epub3/ ), "Readium for Chrome" refers to the Chrome "app" (formerly Chrome "extension") which is now obsolete due to Google's policy. However, this app shares the majority of its code with the "cloud/web reader", and therefore offers roughly the same functionality. Here is a link to the codebase for both the Chrome app and the cloud/web reader:
https://github.com/readium/readium-js-viewer

There are probably other reading systems in `epubtest.org` that rely internally on parts of the ReadiumSDK, but they do not necessarily use the latest version / revision of the source code, and they may also disable/enable some features.

Are you looking for specific feature support? (e.g. popup footnotes)

**Comment by sirespin** [comment]
Hello, thanks for answering.

I was really looking for a list of functionalities, similar to epubtest.

I will build on that list, although it corresponds to version 0.12, and the current 0.32 will be many differences.

Thank you

**Comment by danielweck** [comment]
I do not think that the increased version number reflects a significant evolution in terms of support for EPUB3 features, more like numerous "under the hood" improvements. I guess a notable added feature would have been the "user-selectable font", but this is not an EPUB3 feature per-se.

https://readium.org/development/releases/
